salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -29,21 +31,33 @@ Install containerd:
     - require:
       - test: Repositories configured
       - metalk8s_package_manager: Install runc
+      - file: Create containerd service drop-in
+      - file: Configure registry IP in containerd conf
```

Comment:
For me it's not intuitive to do like that because that mean on the first install you will do in this order
```
- Configure repo
- Install runc
- Create containerd service file
- Create containerd config file toml
- Install containerd
[...]
- Restart containerd <--- Because the `service.running` in "configured.sls" watch containerd service file and containerd config toml
```
And btw you really want to have this 2 "watch" because if containerd already installed and you change something in one of this two file you want the state to restart containerd
 
For me this two state should be in "configured.sls"

Reply:
IIRC, containerd is not automatically started after its installation, that's the goal of this.
You ensure that you only start it when everything is ready.